src/docs/swagger.js

```diff
@@ -26,11 +26,15 @@ export default {
   tags: [
     {
       name: 'users',
-      description: 'The users of Author\'s Haven'
+      description: "The users of Author's Haven"
     },
     {
       name: 'articles',
-      description: 'The articles created by Author\'s Haven users'
+      description: "The articles created by Author's Haven users"
```

Comment:
Juwon, I think you should change this to a single quote.

Reply:
Thanks for the observation, that was a mistake on my path. I'll fix that right away